packages/pearl-diver-react-native/ios/PearlDiver.m

```diff
@@ -27,7 +27,7 @@ @implementation PearlDiver
 }
 
 // PoW
-RCT_EXPORT_METHOD(doPoW:(NSString *)trytes minWeightMagnitude:(int)minWeightMagnitude resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
+RCT_EXPORT_METHOD(doPOW:(NSString *)trytes minWeightMagnitude:(int)minWeightMagnitude resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
```

Comment:
Fixed the method name here so that it would match https://github.com/chrisdukakis/iota.js/blob/feat/pearl-diver-react-native/packages/pearl-diver-react-native/src/index.ts#L45